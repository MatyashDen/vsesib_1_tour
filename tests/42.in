9 9 9 9
0 1
1 7
2 6
3 0
3 7
4 3
4 7
5 6
6 2
