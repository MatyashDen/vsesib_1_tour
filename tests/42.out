104
0RLE0LRE0DUE0RRLLE0RDULE0LDURE0DDUUE0RRRLLLE0RRDULLE0RDDUULEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
1RLE1LRE1DUE1UDE1RDULE1RUDLE1LLRRE1LUDRE1LLLRRRE1LLUDRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2LLRRE2LDUREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
3RLE3DUE3UDE3RRLLE3RDULE3DDUUEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
4RLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
5RLE5LRE5DUE5UDE5RRLLE5RDULE5RUDLE5UUDDEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
6RLE6LRE6DUE6RDULE6DDUUE6RDDUULE6DDDUUUE6RDDDUUULE6DDDDUUUUE6RDDDDUUUULEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
7LRE7DUE7LDURE7DDUUE7LDDUURE7DDDUUUE7LDDDUUUREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
8RLE8LRE8DUE8UDE8RRLLE8RDULE8LLRRE8LDURE8LUDRE8DDUUE8RRDULLE8RDDUULE8LLDURRE8LDDUURE8RRDDUULLE8LLDDUURRE
